### PR TITLE
GitHub actions integration tests

### PR DESCRIPTION
**Pull request by ChristophWurst**
- [x] Integration tests
- [x] Coverage report
- [x] Packaging test

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -48,9 +48,8 @@ jobs:
       name: php${{ matrix.php-versions }}-${{ matrix.db }} integration tests
       services:
           mail-service:
-              image: christophwurst/imap-devel
+              image: antespi/docker-imap-devel
               env:
-                  HOSTNAME: mail.domain.tld
```
Comment (ChristophWurst) [inline .github/workflows/test.yml]: :heavy_check_mark:

**Comment by ChristophWurst** [comment]
> C: 3 AUTHENTICATE PLAIN [INITIAL CLIENT RESPONSE (username: user@domain.tld)]
> S: 3 NO [UNAVAILABLE] Maximum number of connections from user+IP exceeded (mail_max_userip_connections=10)

almost there :pray:

**Review by kesselb — APPROVED** [review]
Nice one :tada:

**Review by rullzer — APPROVED** [review]
whee!

**Review by st3iny — APPROVED** [review]
epic

**Pull request merged**